test/testcafe/spec/IdentifyOktaVerify_spec.js

```diff
@@ -52,8 +54,9 @@ test('should the correct title', async t => {
 test('should the correct content', async t => {
   const identityPage = await setup(t);
   await t.expect(identityPage.getFormTitle()).eql('Sign In');
-  await t.expect(identityPage.getOktaVerifyButtonText()).eql('Sign in with Okta FastPass');
-  await t.expect(identityPage.getSeparationLineText()).eql('or');
+  await t.expect(identityPage.getOktaVerifyButtonText()).contains('Sign in with Okta FastPass');
```

Comment:
Changes `.eql` → `.contains`: what else is in the text? i.e., what caused the original assertion to fail?